src/main/java/io/github/onecx/product/store/bff/rs/controllers/MicrofrontendsRestController.java

```diff
@@ -0,0 +1,113 @@
+package io.github.onecx.product.store.bff.rs.controllers;
+
+import jakarta.enterprise.context.ApplicationScoped;
+import jakarta.inject.Inject;
+import jakarta.transaction.Transactional;
+import jakarta.validation.ConstraintViolationException;
+import jakarta.ws.rs.WebApplicationException;
+import jakarta.ws.rs.core.Response;
+
+import org.eclipse.microprofile.rest.client.inject.RestClient;
+import org.jboss.resteasy.reactive.RestResponse;
+import org.jboss.resteasy.reactive.server.ServerExceptionMapper;
+import org.tkit.quarkus.log.cdi.LogService;
+
+import gen.io.github.onecx.product.store.bff.clients.api.MicrofrontendsInternalApi;
+import gen.io.github.onecx.product.store.bff.clients.model.Microfrontend;
+import gen.io.github.onecx.product.store.bff.clients.model.MicrofrontendPageResult;
+import gen.io.github.onecx.product.store.bff.clients.model.ProblemDetailResponse;
+import gen.io.github.onecx.product.store.bff.rs.internal.MicrofrontendsApiService;
+import gen.io.github.onecx.product.store.bff.rs.internal.model.*;
+import io.github.onecx.product.store.bff.rs.mappers.ExceptionMapper;
+import io.github.onecx.product.store.bff.rs.mappers.MicrofrontendsMapper;
+import io.github.onecx.product.store.bff.rs.mappers.ProblemDetailMapper;
+
+@LogService
+@ApplicationScoped
+@Transactional(value = Transactional.TxType.NOT_SUPPORTED)
+public class MicrofrontendsRestController implements MicrofrontendsApiService {
+
+    @RestClient
+    @Inject
+    MicrofrontendsInternalApi client;
+
+    private final MicrofrontendsMapper mapper;
+
+    private final ProblemDetailMapper problemDetailMapper;
+
+    private final ExceptionMapper exceptionMapper;
+
+    @Inject
+    public MicrofrontendsRestController(MicrofrontendsMapper mapper,
+            ProblemDetailMapper problemDetailMapper, ExceptionMapper exceptionMapper) {
+
+        this.mapper = mapper;
+        this.problemDetailMapper = problemDetailMapper;
+        this.exceptionMapper = exceptionMapper;
+    }
+
+    @Override
+    public Response createMicrofrontend(CreateMicrofrontendRequestDTO createMicrofrontendRequestDTO) {
+
+        try (Response response = client.createMicrofrontend(mapper.mapCreateMfe(createMicrofrontendRequestDTO))) {
+            Microfrontend createdMfe = response.readEntity(Microfrontend.class);
+            MicrofrontendDTO createdMfeDTO = mapper.mapMfe(createdMfe);
+            return Response.status(response.getStatus()).entity(createdMfeDTO).build();
+        } catch (WebApplicationException ex) {
```

Comment:
You probably don't need a catch block here, because the only possbile error case would be a Bad Request which is already covered by the @ServerExceptionMapper. 
As far as I know, the only case where we need to add a catch block is when a 404 is possble (getById, Update etc..)

Reply:
Is it possbile, that 404s are also already covered by your "restException" method?

Reply:
necessary, othterwise data from the response (problemdetails) is missed and not returned. Nevertheless, I removed it from the GET endpoints (product, mfe).